content/en/docs/Features/validations.md

```diff
@@ -7,6 +7,19 @@ Kiali performs a set of validations on your Istio Objects, such as Destination R
 
 ![Istio Config Validation](/images/documentation/features/istio-config-validation.png "Istio Config Validation")
 
+## Disabling validations
+
+In certain environments, particularly those with a high volume of configurations or limited resources, the Istio validation process can be time and resource intensive, potentially causing delays. To prioritize speed and resource efficiency in such scenarios, Kiali offers the option to disable these validations by configuring the validation_reconcile_interval setting to "0s" within the Kiali CR.
+
+Below is an example of a Kiali Operator CR with validations disabled:
```

Comment:
We don't call this the "Kiali Operator CR" -- docs always refer to it as "Kiali CR"
```suggestion
Below is an example of a Kiali CR with validations disabled:
```

Reply:
Right, I thought I corrected that, sorry.